### DISCUSSION TITLE
Can I use the topic operator on a Kafka already installed without using Strimzi/Operator?

### DISCUSSION THREAD

**Discussion by SebTardif** [strimzi/strimzi-kafka-operator#6953] [Q&A]
How do I define the information about that existing cluster so that the topic operator knows what to do?

**Comment by scholzj** [comment]
The Topic Operator deployment has environment variables on to configure the connection to the Kafka cluster (you can check the docs => the standalone topic operator installation for more details). And the topics are created in this cluster.
